src/core/Microsoft.PowerToys.Settings.UI/Package.appxmanifest

```diff
@@ -9,7 +9,7 @@
   <Identity
     Name="f4f787a5-f0ae-47a9-be89-5408b1dd2b47"
     Publisher="CN=lamotile"
-    Version="0.20.0.0" />
+    Version="##VERSION.0" />
```

Comment:
This file should not even be present in the repo, it's used to create the MSIX for this project.
When building using VS it doesn't get created, but when building in CI it does.
We should open an issue to fix this project to not have a Package definition in the first place.